addons/sale_crm/wizard/crm_opportunity_to_quotation_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="crm_quotation_partner_view_form" model="ir.ui.view">
+        <field name="name">crm.quotation.partner.view.form</field>
+        <field name="model">crm.quotation.partner</field>
+        <field name="arch" type="xml">
+            <form string="New Quotation">
+                <group>
+                    <group>
+                        <field name="action" widget="radio"/>
```

Comment:
the lead field should be in the view (hidden), otherwise it will not be in list_field of default_get

Reply:
done